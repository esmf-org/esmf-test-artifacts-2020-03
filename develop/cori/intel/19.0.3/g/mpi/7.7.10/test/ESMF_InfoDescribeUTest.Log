build time -- 2022-02-13 02:56:48.807888
20220213 082344.875 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220213 082344.875 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220213 082344.875 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220213 082344.875 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220213 082344.875 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220213 082344.875 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220213 082344.875 INFO             PET0 Running with ESMF Version   : ESMF_8_3_0_beta_snapshot_06-14-g9f69d9060c
20220213 082344.875 INFO             PET0 ESMF library build date/time: "Feb 13 2022" "02:53:12"
20220213 082344.875 INFO             PET0 ESMF library build location : /global/u2/r/rsdunlap/esmf-testing/intel_19.0.3_mpi_g_develop
20220213 082344.875 INFO             PET0 ESMF_COMM                   : mpi
20220213 082344.881 INFO             PET0 ESMF_MOAB                   : enabled
20220213 082344.881 INFO             PET0 ESMF_LAPACK                 : enabled
20220213 082344.881 INFO             PET0 ESMF_NETCDF                 : enabled
20220213 082344.881 INFO             PET0 ESMF_PNETCDF                : disabled
20220213 082344.881 INFO             PET0 ESMF_PIO                    : enabled
20220213 082344.881 INFO             PET0 ESMF_YAMLCPP                : enabled
20220213 082344.883 INFO             PET0  Beginning Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 51
20220213 082344.883 INFO             PET0  NUMBER_OF_PROCESSORS           4
20220213 082344.884 ERROR            PET0 ESMF_InfoDescribe.F90:146 ESMF_InfoDescribeInitialize() Failure  - Object already initialized
20220213 082344.884 INFO             PET0  PASS  No double initialization, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 71
20220213 082344.884 INFO             PET0  PASS  Initialize/Destroy, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 85
20220213 082344.885 INFO             PET0  PASS  Initialize behavior, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 110
20220213 082344.885 INFO             PET0  PASS  Mesh is enabled, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 146
20220213 082344.885 INFO             PET0  Ending Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 153
20220213 082344.886 INFO             PET0 Finalizing ESMF
20220213 082344.875 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220213 082344.875 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220213 082344.875 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220213 082344.875 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220213 082344.875 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220213 082344.875 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220213 082344.875 INFO             PET1 Running with ESMF Version   : ESMF_8_3_0_beta_snapshot_06-14-g9f69d9060c
20220213 082344.875 INFO             PET1 ESMF library build date/time: "Feb 13 2022" "02:53:12"
20220213 082344.875 INFO             PET1 ESMF library build location : /global/u2/r/rsdunlap/esmf-testing/intel_19.0.3_mpi_g_develop
20220213 082344.875 INFO             PET1 ESMF_COMM                   : mpi
20220213 082344.878 INFO             PET1 ESMF_MOAB                   : enabled
20220213 082344.878 INFO             PET1 ESMF_LAPACK                 : enabled
20220213 082344.878 INFO             PET1 ESMF_NETCDF                 : enabled
20220213 082344.878 INFO             PET1 ESMF_PNETCDF                : disabled
20220213 082344.878 INFO             PET1 ESMF_PIO                    : enabled
20220213 082344.878 INFO             PET1 ESMF_YAMLCPP                : enabled
20220213 082344.883 INFO             PET1  Beginning Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 51
20220213 082344.883 INFO             PET1  NUMBER_OF_PROCESSORS           4
20220213 082344.883 ERROR            PET1 ESMF_InfoDescribe.F90:146 ESMF_InfoDescribeInitialize() Failure  - Object already initialized
20220213 082344.883 INFO             PET1  PASS  No double initialization, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 71
20220213 082344.884 INFO             PET1  PASS  Initialize/Destroy, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 85
20220213 082344.884 INFO             PET1  PASS  Initialize behavior, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 110
20220213 082344.884 INFO             PET1  PASS  Mesh is enabled, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 146
20220213 082344.885 INFO             PET1  Ending Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 153
20220213 082344.885 INFO             PET1 Finalizing ESMF
20220213 082344.875 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220213 082344.875 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220213 082344.875 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220213 082344.875 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220213 082344.875 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220213 082344.875 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220213 082344.875 INFO             PET2 Running with ESMF Version   : ESMF_8_3_0_beta_snapshot_06-14-g9f69d9060c
20220213 082344.875 INFO             PET2 ESMF library build date/time: "Feb 13 2022" "02:53:12"
20220213 082344.875 INFO             PET2 ESMF library build location : /global/u2/r/rsdunlap/esmf-testing/intel_19.0.3_mpi_g_develop
20220213 082344.875 INFO             PET2 ESMF_COMM                   : mpi
20220213 082344.880 INFO             PET2 ESMF_MOAB                   : enabled
20220213 082344.880 INFO             PET2 ESMF_LAPACK                 : enabled
20220213 082344.880 INFO             PET2 ESMF_NETCDF                 : enabled
20220213 082344.880 INFO             PET2 ESMF_PNETCDF                : disabled
20220213 082344.880 INFO             PET2 ESMF_PIO                    : enabled
20220213 082344.880 INFO             PET2 ESMF_YAMLCPP                : enabled
20220213 082344.883 INFO             PET2  Beginning Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 51
20220213 082344.883 INFO             PET2  NUMBER_OF_PROCESSORS           4
20220213 082344.883 ERROR            PET2 ESMF_InfoDescribe.F90:146 ESMF_InfoDescribeInitialize() Failure  - Object already initialized
20220213 082344.884 INFO             PET2  PASS  No double initialization, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 71
20220213 082344.884 INFO             PET2  PASS  Initialize/Destroy, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 85
20220213 082344.884 INFO             PET2  PASS  Initialize behavior, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 110
20220213 082344.885 INFO             PET2  PASS  Mesh is enabled, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 146
20220213 082344.885 INFO             PET2  Ending Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 153
20220213 082344.885 INFO             PET2 Finalizing ESMF
20220213 082344.875 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220213 082344.875 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220213 082344.875 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220213 082344.875 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220213 082344.875 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220213 082344.875 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220213 082344.875 INFO             PET3 Running with ESMF Version   : ESMF_8_3_0_beta_snapshot_06-14-g9f69d9060c
20220213 082344.875 INFO             PET3 ESMF library build date/time: "Feb 13 2022" "02:53:12"
20220213 082344.875 INFO             PET3 ESMF library build location : /global/u2/r/rsdunlap/esmf-testing/intel_19.0.3_mpi_g_develop
20220213 082344.875 INFO             PET3 ESMF_COMM                   : mpi
20220213 082344.878 INFO             PET3 ESMF_MOAB                   : enabled
20220213 082344.878 INFO             PET3 ESMF_LAPACK                 : enabled
20220213 082344.878 INFO             PET3 ESMF_NETCDF                 : enabled
20220213 082344.878 INFO             PET3 ESMF_PNETCDF                : disabled
20220213 082344.878 INFO             PET3 ESMF_PIO                    : enabled
20220213 082344.878 INFO             PET3 ESMF_YAMLCPP                : enabled
20220213 082344.883 INFO             PET3  Beginning Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 51
20220213 082344.884 INFO             PET3  NUMBER_OF_PROCESSORS           4
20220213 082344.884 ERROR            PET3 ESMF_InfoDescribe.F90:146 ESMF_InfoDescribeInitialize() Failure  - Object already initialized
20220213 082344.884 INFO             PET3  PASS  No double initialization, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 71
20220213 082344.884 INFO             PET3  PASS  Initialize/Destroy, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 85
20220213 082344.885 INFO             PET3  PASS  Initialize behavior, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 110
20220213 082344.885 INFO             PET3  PASS  Mesh is enabled, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 146
20220213 082344.885 INFO             PET3  Ending Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 153
20220213 082344.885 INFO             PET3 Finalizing ESMF
