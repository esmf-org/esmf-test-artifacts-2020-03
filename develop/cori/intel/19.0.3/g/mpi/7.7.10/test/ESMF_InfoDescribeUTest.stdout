build time -- 2022-01-23 06:21:14.604375
  Beginning Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F
 90, line 51
  Beginning Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F
 90, line 51
  Beginning Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F
 90, line 51
  Beginning Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F
 90, line 51
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
  PASS  No double initialization, src/Superstructure/InfoAPI/tests/ESMF_InfoDesc
 ribeUTest.F90, line 71
  PASS  No double initialization, src/Superstructure/InfoAPI/tests/ESMF_InfoDesc
 ribeUTest.F90, line 71
  PASS  No double initialization, src/Superstructure/InfoAPI/tests/ESMF_InfoDesc
 ribeUTest.F90, line 71
  PASS  No double initialization, src/Superstructure/InfoAPI/tests/ESMF_InfoDesc
 ribeUTest.F90, line 71
  PASS  Initialize/Destroy, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUT
 est.F90, line 85
  PASS  Initialize/Destroy, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUT
 est.F90, line 85
  PASS  Initialize/Destroy, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUT
 est.F90, line 85
  PASS  Initialize/Destroy, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUT
 est.F90, line 85
  PASS  Initialize behavior, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeU
 Test.F90, line 110
  PASS  Initialize behavior, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeU
 Test.F90, line 110
  PASS  Initialize behavior, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeU
  PASS  Initialize behavior, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeU
 Test.F90, line 110
 Test.F90, line 110
  PASS  Mesh is enabled, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest
 .F90, line 146
  PASS  Mesh is enabled, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest
 .F90, line 146
  PASS  Mesh is enabled, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest
 .F90, line 146
  PASS  Mesh is enabled, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest
 .F90, line 146
  Ending Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90,
  line 153
  Ending Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90,
  line 153
  Ending Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90,
  line 153
  Ending Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90,
  line 153
  PET           0  Test Elapsed Time    5.24300000000000       msec.
  PET           1  Test Elapsed Time    42.1840000000000       msec.
  PET           2  Test Elapsed Time    42.0420000000000       msec.
  PET           3  Test Elapsed Time    42.2000000000000       msec.
